common/djangoapps/util/enterprise_helpers.py

```diff
@@ -225,3 +236,63 @@ def get_enterprise_branding_filter_param(request):
 
     """
     return request.session.get(ENTERPRISE_CUSTOMER_BRANDING_OVERRIDE_DETAILS, None)
+
+
+def get_dashboard_consent_notification(request, user, course_enrollments):
+    """
+    If relevant to the request at hand, create a banner on the dashboard indicating consent failed.
```

Comment:
Please describe the Args and Returns characteristics of this function in the docstring.